tox.ini

```diff
@@ -132,6 +136,7 @@ install_command =
     pip install {opts} {packages} --isolated --upgrade --force-reinstall
```

Comment:
Are constraints not used here on purpose? Why not just inherit this command from the testenv like all the others?

Reply:
I think so.